output/cloud/expv2/hdr.go

```diff
@@ -20,9 +25,10 @@ const (
 
 	// highestTrackable represents the maximum
 	// value that the histogram is able to track with high accuracy (0.1% of error).
-	// It should be a high enough
-	// and rationale value for the k6 context; 2^30 = 1_073_741_824
-	highestTrackable = 1 << 30
+	// It represents the maximum number for int64 using the
+	// default minimum resolution, where it multiplies the value ingested from the histogram
+	// of a specific factor. Essentially, having higher resolution reduces the highest limit.
+	highestTrackable = math.MaxInt64 * defaultMinimumResolution
```

Comment:
Perhaps I am missing something but highest trackable value is checked after dividing by resolution `v /= h.MinimumResolution`, and so resolution should not be mentioned here.

```suggestion
	highestTrackable = math.MaxInt64
```

Reply:
This is now expanded to the highest, good catch.